arch/x86/kvm/vmx/pkvm/hyp/io_emulate.h

```diff
@@ -32,6 +32,8 @@ struct pkvm_pio_emul_table {
 	DECLARE_BITMAP(bitmap, PKVM_MAX_PIO_EMUL_NUM);
 };
 
+int register_host_pio_handler(struct pkvm_host_vm *host_vm, unsigned int port,
+	unsigned int size_mask, pio_handler_t read, pio_handler_t write);
```

Comment:
we may expose this at first patch it come out.

Reply:
I actually did this. Static in [pkvm: introduce port I/O emulation framework](https://github.com/intel-staging/pKVM-IA/pull/22/commits/8f6f66622a61328621267177fb73fde699c3f6f9) and exposed in [pkvm: emulate PCI config space access](https://github.com/intel-staging/pKVM-IA/pull/22/commits/e232085fc6152e8499804d4ed7108f704fdac219).

Reply:
what I mean is you can expose it in [pkvm: introduce port I/O emulation framework](https://github.com/intel-staging/pKVM-IA/pull/22/commits/8f6f66622a61328621267177fb73fde699c3f6f9), but I am ok if you keep current solution.

Reply:
Got it, I will change.

Reply:
Fixed it.